integration/singlearch/fixtures/test-images.json

```diff
@@ -14,15 +15,17 @@
         "digest": "sha256:42d3e6bc186572245aded5a0be381012adba6d89355fa9486dd81b0c634695b5",
         "distro": "Alpine",
         "description": "Valid apk/db, apk present, locally tagged with fully-qualified name",
-        "ignoreErrors": false
+        "ignoreErrors": false,
+        "isManifest": false
     },
     {
         "image": "docker.io/library/nginx",
         "tag": "1.21.6",
         "digest": "sha256:2bcabc23b45489fb0885d69a06ba1d648aeda973fae7bb981bafbb884165e514",
         "distro": "Debian",
         "description": "Valid dpkg/status, apt present",
-        "ignoreErrors": false
+        "ignoreErrors": false,
+        "isManifest": true
```

Comment:
this is a manifest list, why do we mark as manifest?

Reply:
do we mean `isManifestList` here?

Reply:
yes, should be isManifestList. Updated this!